src/shell/lisa_shell

```diff
@@ -256,7 +256,7 @@ EOF
 
 function _lisa-test {
 CLASS=${1}
-nosetests -v tests/$CLASS
+nosetests --nocapture --nologcapture -v tests/$CLASS
```

Comment:
Up to you.  One of the problems that I had with the way we are doing logging right now is that if the first test of a testcase fails, then all the output from the test up to the point of failure is added as "captured stdout", including board and test set up.  If you have multiple test failures, it is very hard to see them all and differentiate the original stdout from the duplicated one.

Maybe what we should do is to remove the reloading of the logging module in the `Executor`.